packages/atlas/lib/src/traffic_level.dart

```diff
@@ -0,0 +1,10 @@
+enum TrafficLevel {
```

Comment:
Can we rename to this to congestion level

Reply:
Sure! I already changed the name.